samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
```

Comment:
Can we fix the alignment for image ?
![image](https://user-images.githubusercontent.com/59052547/184617972-760c7e51-27e5-4598-8677-1f174af34f52.png)

Reply:
Also, we could have provided proper app name
![image](https://user-images.githubusercontent.com/59052547/184618314-28c32351-a071-4553-be34-e484f1fda66b.png)

Reply:
Fixed alignment by moving inputs to next line